src/js/components/Hero.js

```diff
@@ -0,0 +1,148 @@
+// (C) Copyright 2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import CSSClassnames from '../utils/CSSClassnames';
+import Box from 'grommet/components/Box';
+import Image from 'grommet/components/Image';
+import Video from 'grommet/components/Video';
+
+const CLASS_ROOT = CSSClassnames.HERO;
+const LIGHT_COLORINDEX = 'light-1';
+const DARK_COLORINDEX = 'grey-1';
+const PALM_BREAKPOINT = 720;
```

Comment:
Can we use utils/Responsive as the single source for this, rather than hard-coding the breakpoint in this file?